whitepaper/whitepaper.md

```diff
@@ -117,6 +117,9 @@ Photo by Pontus Wellgraf on Unsplash
     - [5.6.3 Key/Value Storage Plugins](#563-keyvalue-storage-plugins)
     - [5.6.4 Serverside Keychain Plugins](#564-serverside-keychain-plugins)
     - [5.6.5 Manual Consortium Plugin](#565-manual-consortium-plugin)
+  - [5.7 Deployment Scenarios](#57-deployment-scenarios)
```

Comment:
I have concerns about the fragmentation of architectural descriptions. This kind of information should be part of architecture 5.1 when the content is agreed upon among contributors.

Reply:
> I have concerns about the fragmentation of architectural descriptions. This kind of information should be part of architecture 5.1 when the content is agreed upon among contributors.

@sfuji822 Did you mean like this? 6a53a97fa488c0dc17f8d5e43d2f3c18e8ac3fb8

Reply:
> @sfuji822 Did you mean like this? [6a53a97](https://github.com/hyperledger/cactus/commit/6a53a97fa488c0dc17f8d5e43d2f3c18e8ac3fb8)

@petermetz  I think that is close, but I suggest to make it more simple structured one.
The title ("5.1 Integration patterns") will be changed to "Deployment Scinarios" since existing 5.1 is describing possible integration patterns with asumed deployment and can be merged.
I thought 5.1.1 can be texts which following existing 5.1 texts. 
I am not sure we still need to subsection of examples.
Those texts(examples) can follow 5.1.1 text since there are not normative specification in adding contents.

Reply:
> The title ("5.1 Integration patterns") will be changed to "Deployment Scinarios" since existing 5.1 is describing possible integration patterns with asumed deployment and can be merged.
> I thought 5.1.1 can be texts which following existing 5.1 texts.

@sfuji822 Done: 1535dbedabe6195429ef6c4510da75f5e6a9f0a1

> I am not sure we still need to subsection of examples.

@sfuji822 Hard to be sure of course. It's a subjective question, I agree. What is an objective fact though: I put those there based on specific requests from the community.

> Those texts(examples) can follow 5.1.1 text since there are not normative specification in adding contents.

@sfuji822 Not sure I'm getting this. Do you mean that the section titles could be downgraded to just bold text maybe?
Like going from `### 5.1.1 Production Deployment Example` to `**Production Deployment Example**`?

Reply:
I can go with as is since we agreed to revise chapter 5 later.